OJP/OJP_Trips.xsd

```diff
@@ -414,6 +422,7 @@
 					<xs:documentation>Geographic embedding of this leg.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:group ref="SustainabilityGroup" minOccurs="0"/>
```

Comment:
Why don't you put it at TripLegStructure level so it is available for all ?

Reply:
I moved it. TransferLegs hopefully are quite sustainable. But it does not hurt.